packages/protocol/contracts/identity/Escrow.sol

```diff
@@ -246,6 +348,176 @@ contract Escrow is
     return sentPaymentIds[sender];
   }
 
+  /**
+  * @notice Gets array of all trusted issuers set per paymentId.
+  * @param paymentId The ID of the payment to be deleted.
+  * @return An array of addresses of trusted issuers set for an escrowed payment.
+  */
+  function getTrustedIssuersPerPayment(address paymentId) external view returns (address[] memory) {
+    return trustedIssuersPerPayment[paymentId];
+  }
+
+  /**
+  * @notice Gets trusted issuers set as default for payments by `transfer` function.
+  * @return An array of addresses of trusted issuers.
+  */
+  function getDefaultTrustedIssuers() public view returns (address[] memory) {
+    return defaultTrustedIssuers;
+  }
```

Comment:
yeah, added these getters (here and above) to make unit testing a bit easier to read/cleaner (just to get all values all at once instead of needing to iterate through the arrays in the tests). Do you think that's an ok reason to keep or is it still preferred to remove and just have a length getter?